src/views/Books/books.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+import { fetchBooks } from '../../services/books';
+
+export default function Books() {
+  const [books, setBooks] = useState([]);
+
+  useEffect(() => {
+    //fetchBooks page-load
+    const fetchData = async () => {
+      const data = await fetchBooks();
+      setBooks(data);
+      // console.log('book data', data);
+    };
+    fetchData();
```

Comment:
For error handling, we can add a try/catch block to catch errors on our third party APIs.